src/transformers/visitors/toContractVisitor.ts

```diff
@@ -69,6 +64,15 @@ const findCustomInputsVisitor = (thisPath: NodePath, thisState: any) => {
   }
 };
 
+
+function parentnewASTPointer(parent , path , newNode){
```

Comment:
This is a great addition, thank you! I know for orchestration sometimes we need to look at `path.containerName` and sometimes just `parent._newASTPointer`, so would it be possible to also have this extracted there and for circuits? If it's too complex, no worries

Reply:
Yeah . I agree . I will create similar one for circuits as well . Thanks for suggestion.